Сделать migrate.js идемпотентным

Сейчас при выкатке мы вызываем `src/migrate.js` 
Он падает с логом:

```
> hakeshonassybot@1.0.0 migrate /app
> node src/migrate.js
Start migration...
(node:42) DeprecationWarning: current Server Discovery and Monitoring engine is deprecated, and will be removed in a future version. To use the new Server Discover and Monitoring engine, pass option { useUnifiedTopology: true } to the MongoClient constructor.
migration failed with error:  MongoError: a collection 'hakeshonassydb_production.messages' already exists
    at Connection.<anonymous> (/app/node_modules/mongodb/lib/core/connection/pool.js:451:61)
    at Connection.emit (events.js:314:20)
    at processMessage (/app/node_modules/mongodb/lib/core/connection/connection.js:452:10)
    at TLSSocket.<anonymous> (/app/node_modules/mongodb/lib/core/connection/connection.js:621:15)
    at TLSSocket.emit (events.js:314:20)
    at addChunk (_stream_readable.js:298:12)
    at readableAddChunk (_stream_readable.js:273:9)
    at TLSSocket.Readable.push (_stream_readable.js:214:10)
    at TLSWrap.onStreamRead (internal/stream_base_commons.js:188:23) {
  operationTime: Timestamp { _bsontype: 'Timestamp', low_: 8, high_: 1603202328 },
  ok: 0,
  code: 48,
  codeName: 'NamespaceExists',
  '$clusterTime': {
    clusterTime: Timestamp { _bsontype: 'Timestamp', low_: 8, high_: 1603202328 },
    signature: { hash: [Binary], keyId: [Long] }
  }
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! hakeshonassybot@1.0.0 migrate: `node src/migrate.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the hakeshonassybot@1.0.0 migrate script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /app/.npm/_logs/2020-10-20T13_58_49_265Z-debug.log

```

Нужно сделать выполнение идемпотентным, то есть если у нас нет коллекции и индексов, создаём их, если есть то не создаём но и не падаем.

--- a/src/migrate.js
+++ b/src/migrate.js
@@ -7,17 +7,12 @@ const createDbCollection = async () => {
   const mongo = new MongoClient(url);
   const client = await mongo.connect();
 
-  await client.db(dbName).createCollection('messages');
-
-  await client
-    .db(dbName)
-    .collection('messages')
-    .createIndex({ 'chat.id': 1 });
-
-  await client
-    .db(dbName)
-    .collection('messages')
-    .createIndex({ date: -1 });
+  let messages = await client.db(dbName).collection('messages');
+  if (!messages) {
+    messages = await client.db(dbName).createCollection('messages');
+  }
+  await messages.createIndex({ 'chat.id': 1 });
+  await messages.createIndex({ date: -1 });
 };
 
 console.log('Start migration...');

